citation/management/commands/export_data.py

```diff
@@ -1,9 +1,8 @@
 from django.core.management.base import BaseCommand
 from django.db.models import Count
 
-from citation.models import Publication
+from citation.models import Publication, Note, Platform, Sponsor
 
-import unicodecsv as csv
```

Comment:
this is good - we should replace unicodecsv with the native Python 3 [csv module](https://docs.python.org/3/library/csv.html)